server/server.js

```diff
@@ -153,7 +153,14 @@ app.post(`${apiBaseUrl}/approve-user`,
   keycloak.allowRolesMiddleware('ministry_of_health'),
   asyncMiddleware(async (req, res) => {
     await validate(AccessRequestApproval, req.body);
-    await keycloak.approvePendingRequest(req.body.userId, req.body.role, [req.body.region]);
+    const promises = req.body.sites.map((siteId) => dbClient.db[collections.EMPLOYER_SITES].findDoc({ siteId }));
```

Comment:
Is the only way to do this via multiple `findDoc` calls? There's no way to query by array?